docs/commerce/checkout/checkout_api.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Use PHP API to work with checkouts in Commerce.
+edition: commerce
+---
+
+# Checkout API
+
+To get checkouts and manage them, use the `\Ibexa\Contracts\Checkout\CheckouterviceInterface` interface.
+
+To manipulate checkouts that are stored in sessions, you use `CheckoutService`. 
+Checkouts are containers for the `Ibexa\Contracts\Cart\Value\CartInterface` object 
+and all the data provided at each step of the [configurable checkout process](configure_checkout.md). 
+The dataset is flexible but minimized due to session size restrictions. 
```

Comment:
I'm wondering if we need this information from the point of view of the user, or is it just implementation details.